shared/pkg/analyzer/trivy/trivy.go

```diff
@@ -92,10 +92,10 @@ func (a *Analyzer) Run(sourceType utils.SourceType, userInput string) error {
 				Scanners: []trivyTypes.Scanner{}, // Disable all security checks for SBOM only scan
 			},
 			ReportOptions: trivyFlag.ReportOptions{
-				Format:       "cyclonedx", // Cyconedx format for SBOM so that we don't need to convert
-				ReportFormat: "all",       // Full report not just summary
-				Output:       &output,     // Save the output to our local buffer instead of Stdout
-				ListAllPkgs:  true,        // By default Trivy only includes packages with vulnerabilities, for full SBOM set true.
+				Format:       "cyclonedx",     // Cyconedx format for SBOM so that we don't need to convert
+				ReportFormat: "all",           // Full report not just summary
+				Output:       output.String(), // Save the output to our local buffer instead of Stdout
```

Comment:
This not gonna work, it looks like trivy changed it from a report into a buffer to something else. Need to understand how it possible to get the output into a buffer again (or find an alternative, I don't like to create a temp file and write it there but maybe we will not have any other option)